### ISSUE TITLE
failed to run hyper cmd killcontainer with docker 1.13.1 

### ISSUE THREAD

**Issue by chavafg** [01org/cc-oci-runtime#747]
Run tests  with the new changes in hyperstart and #738. Containers still get hanged when running with docker 1.13.1 and with 17.03.

These errors appear in the cc-oci-runtime.log: 
```
2017-03-14T22:29:26.915470Z:10592:cc-oci-runtime:debug:writing message data to proxy socket: {"id":"hyper","data":{"hy
perName":"killcontainer","data":{"container":"5a57a0874ef8b537234566d4a26a9c61c29425872532f443bda0ad03211cc829","signa
l":9,"allProcesses":true}}}
2017-03-14T22:29:26.919742Z:10592:cc-oci-runtime:debug:proxy msg length: 58
2017-03-14T22:29:26.919781Z:10592:cc-oci-runtime:debug:message read from proxy socket: {"success":false,"error":"ERROR
 received from Hyperstart"}
2017-03-14T22:29:26.919841Z:10592:cc-oci-runtime:critical:failed to run hyper cmd killcontainer: {"success":false,"error":"ERROR received from Hyperstart"}
2017-03-14T22:29:26.919917Z:10592:cc-oci-runtime:critical:failed to run cmd killcontainer
2017-03-14T22:29:26.919969Z:10592:cc-oci-runtime:debug:disconnecting from proxy
2017-03-14T22:29:26.920029Z:10592:cc-oci-runtime:critical:failed to kill container 5a57a0874ef8b537234566d4a26a9c61c29425872532f443bda0ad03211cc829: Bad file descriptor
2017-03-14T22:29:26.920244Z:10592:cc-oci-runtime:debug:created state file /var/run/cc-oci-runtime/5a57a0874ef8b537234566d4a26a9c61c29425872532f443bda0ad03211cc829/state.json

```

**Comment by chavafg** [comment]
Attaching all log:
[cc-oci-runtime.txt](https://github.com/01org/cc-oci-runtime/files/842951/cc-oci-runtime.txt)

**Comment by devimc** [comment]
fixed in #776
